pkg/proxy/winkernel/proxier.go

```diff
@@ -599,6 +602,8 @@ func NewProxier(
 		return nil, err
 	}
 
+	sessionAffinitySupported = (hcn.SessionAffinitySupported() == nil)
```

Comment:
Perhaps we can make this a member of Proxier, to avoid global variables

Reply:
What do you think of the change I just made? I added the entire supportedFeatures struct to proxier

Reply:
I think it's great. Definitely the right call.

Reply:
thx